app/assets/javascripts/comments.coffee

```diff
@@ -0,0 +1,14 @@
+$(document).on "page:change", ->
+
+  # Shows and hides the comment form
+  $('#write_comment_repository_btn').unbind('click').on 'click', (event) ->
+      $('#write_comment_form').toggle 400, "swing", ->
+        if $('#write_comment_form').is(':visible')
+          $('#write_comment_repository_btn i').addClass("fa-minus-circle")
+          $('#write_comment_repository_btn i').removeClass("fa-plus-circle")
```

Comment:
This doesn't work. Mainly because it has the `fa-user-plus` class. Therefore, all this adding/removing of classes just doesn't work for me. Note that this would fix one of my complaints in my previous comment,